src/stableset/membership.rs

```diff
@@ -0,0 +1,204 @@
+use std::collections::BTreeSet;
+use std::fmt::Debug;
+
+use super::{
+    stable_set::{Member, StableSet},
+    StableSetMsg,
+};
+use crate::comms::NetworkNode;
+
+pub type Elders = BTreeSet<NetworkNode>;
+
+const ELDER_COUNT: usize = 7;
+
+#[derive(Clone, Debug, Eq, Hash, PartialEq)]
+pub struct Membership {
+    pub stable_set: StableSet,
+}
+
+impl Membership {
+    pub fn new(genesis: &BTreeSet<NetworkNode>) -> Self {
```

Comment:
For me genesis means a single node, but in this case you refer to all the initial nodes?